.gitignore

```diff
@@ -1,3 +1,3 @@
 .DS_Store
 node_modules
```

Comment:
Is there a reason you've removed the out folder from gitignore? This should all be autogenerated.

Reply:
I needed to use it, i was pulling it with npm with a direct link toto my github thuse i needed the compiled version sorry. Wil add that also